proto/01-protoc-base/install.sh

```diff
@@ -23,14 +23,14 @@ python3 -m pip install -q --no-cache-dir --upgrade setuptools
 python3 -m pip install -q --no-cache-dir -r "${__dir}/requirements.txt"
 python3 -m pip list
 
-wget -q "https://github.com/protocolbuffers/protobuf/releases/download/v3.17.3/protoc-3.17.3-linux-x86_64.zip"
-wget -q "https://repo1.maven.org/maven2/io/grpc/protoc-gen-grpc-java/1.36.1/protoc-gen-grpc-java-1.36.1-linux-x86_64.exe"
+wget -q "https://github.com/protocolbuffers/protobuf/releases/download/v21.9/protoc-21.9-linux-x86_64.zip"
```

Comment:
Looks like there's an issue with v21 of protoc: https://github.com/protocolbuffers/protobuf-javascript/issues/127
Recommend downgrading to v20.1 (libprotoc 3.20.1)

Reply:
Thanks for the link, most likely the cause of errors I'm seeing.